section3/exercises/arrays.js

```diff
@@ -79,12 +86,12 @@ The post itself likely has more complex data, but here's one way we can think ab
 var posts = ["image at beach", "holiday party", "adorable puppy", "video of cute baby"];
 
 // YOU DO: Think of a web application you commonly use. Where do you see LISTS utilized, where arrays
-// may be storing data? Come up with 3 examples - they could be from different web applications or 
+// may be storing data? Come up with 3 examples - they could be from different web applications or
 // all from the same one.
-
-// 1:
-// 2:
-// 3:
+//Peloton App: cycling classes available on demand
```

Comment:
Did you do the Cody Valentines XO ride? (had to ask)

Great example of strings for an Array!